src/CellData/DomainContributions.jl

```diff
@@ -93,6 +93,18 @@ function (-)(a::DomainContribution,b::DomainContribution)
   c
 end
 
+function (*)(a::Number,b::DomainContribution)
+  c = copy(b)
+  for (trian,array) in b.dict
+    s = size(get_cell_map(trian))
+    constant_array = Fill(a,s)
+    add_contribution!(c,trian,constant_array,*)
+  end
+  c
+end
```

Comment:
I would implement this in a different way so that you do not need to relax the checks above. Something like:

```julia
function (*)(a::Number,b::DomainContribution)
  c = DomainContribution()
  for (trian,array_old) in b.dict
    s = size(get_cell_map(trian))
    array_new = lazy_map(Broadcast(*),Fill(a,s),array_old)
    add_contribution!(c,trian,array_new)
  end
  c
end
```

Reply:
Done in 05c884e45c9bddac2cb2db061e8fe3fcf882c9af